src/components/site-header/eu-site-header.story.js

```diff
@@ -33,6 +33,7 @@ dataHarmonised.has_menu = true;
 const getArgs = (data) => {
   const defaultArgs = {
     show_language_selector: true,
+    show_custom_action: true,
```

Comment:
custom action should be hidden by default

Reply:
hidden by default now